rstudio/rstudio
# move electron NOTICE info into regular file

### Intent

Addresses #9429

### Approach

Merge NOTICE.electron into NOTICE and delete the electron-only copy.
 
### Automated Tests

N/A

### QA Notes

Electron info should now appear in the open-source components list in Help / About. We have a single NOTICE file used for all products (qt desktop, server, workbench, electron, etc).

### Checklist
